content/swift/concepts/optionals/optionals.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'Optionals'
+Description: 'Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Optionals'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).
+
+## Syntax
+
+In Swift, a non-optional variable can't be set to nil. Rather, the optional type must be defined using the added question mark `?`.
```

Comment:
```suggestion
In Swift, a non-optional variable cannot be set to `nil`. Rather, the optional type must be defined using the added question mark `?`.
```

The second sentence should also account for defining optionals with the alternative syntax:

```pseudo
var maybeInt: Optional<Int>
```